LiteEditor/CodeLiteIDE.project

```diff
@@ -2109,7 +2164,7 @@ resources.cpp: resources.xrc
         <SearchPaths>../sdk/clang/include</SearchPaths>
       </Completion>
     </Configuration>
-    <Configuration Name="Win_x64_Release" CompilerType="g++-64" DebuggerType="GNU gdb debugger" Type="Executable" BuildCmpWithGlobalSettings="append" BuildLnkWithGlobalSettings="append" BuildResWithGlobalSettings="append">
```

Comment:
Can you please rename your local compiler from `MinGW ( mingw64 )` to `g++-64` to 
avoid this change?